content/rs/administering/active-active.md

```diff
@@ -95,54 +74,62 @@ Do you want to set up NTP time synchronization now [Y/N]? Y
-for specifics on what ports should be open for VPNs and Security groups.
-
-### **Data Persistence**
-
-After creating CRDB instances on each participating cluster, you can
-individually set your data persistence configuration. Just like regular
-Redis Enterprise databases, CRDB data persistence options include AOF
-(Append-Only File) data persistence and
-snapshot.
-
-### Syncer process
-
-Each node in a cluster containing a CRDB Instance hosts a process called
-syncer. Its job is to synchronize data from other Instances of the CRDB.
-It connects to the other cluster's proxy and reads data from that
-database, brings it back and then writes it to the appropriate master
-shard of that database.
+## Network Configurations
+
+RS assumes that networking between the clusters is already configured when you create a CRDB.
+For security purposes, recommend that you configure a secure VPN between all clusters that host a CRDB instance.
+The setup of the CRDB fails if there is no connectivity between the clusters.
+
+## Network Ports
+
+For initial configuration and ongoing maintenance of a CRDB, every nude must have access to the REST API ports of every other node.
+You must also open ports for [VPNs and Security groups]({{< relref "/rs/administering/designing-production/networking/port-configurations.md" >}}).
+
+For synchronization, CRDBs operate over the standard endpoint ports.
+The endpoint port that you configure when you create the CRDB is the endpoint port of the proxy for that CRDB on each cluster.
+
+### Data Persistence
+
+You can set the data persistence configuration, including AOF (Append-Only File) data persistence and snapshot,
+for each participating cluster.
+
+## Syncer process
+
+Each node in a cluster containing a CRDB instance hosts a process called syncer.
+The syncer process:
+
+1. Connects to the other cluster proxy
+1. Reads data from that database
+1. Writes the data to the master shard of that database
+
+This process is also used in the [open-source redis replication](https://redis.io/topics/replication).
+
+The Master at the top of the master-slaves tree has a replication ID.
+This replication ID is identical for all slaves in that tree.
+When a new master is appointed, the replication ID changes but a partial sync from the previous ID is still possible.
+In a partial sync, the backlog of operations since the offset are transferred as raw operations.
+In a full sync, the data from the master is transferred to the slave as an RDB file which is followed by a partial sync.
+
+Partial synchronization requires a backlog large enough to store the data operations until connection is restored.
+
+### Syncer in Active-Active Replication (CRDB)
+
+In the case of a CRDB:
+
+- Multiple past replication IDs and offsets are stored to allow for multiple syncs
+- The Active-Active backlog is also sent to the slave during a full sync
+- Full sync triggers heavy data transfers between geo-replicated CRDB instances
```

Comment:
this doesn't feel like it should be a part of the list... also maybe we need to emphasise that it's best to avoid crdt fullsync for that reason